docs/OpenCLCOnVulkan.md

```diff
@@ -33,9 +33,17 @@ The SPIR-V as produced from the OpenCL C language can make use of the following
 capabilities:
 
 - `Shader` as we are targeting the OpenCL C language at a Vulkan implementation.
+- `VariablePointersStorageBuffer`, from the _SPRV\_KHR\_variable\_pointers_ extension.
 - `VariablePointers`, from the _SPV\_KHR\_variable\_pointers_ extension.
-  - *Note*: The compiler always emits code that depends on `VariablePointers`
-    even though there might some cases where it is not strictly needed.
+ - *Note*: the compiler attempts to add the minimal variable pointers capability required.
+- `Int16` if short or ushort types are used.
```

Comment:
What about vectors of these things.  Don't  we go through great lengths to avoid accesses smaller than 32 bit?    In any case, if I'm confused, then others will be.  Best to say something here I think, one way or the other.